go.mod

```diff
@@ -11,8 +11,9 @@ require (
 	github.com/mattn/go-colorable v0.1.4 // indirect
 	github.com/mattn/go-isatty v0.0.12 // indirect
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/mongodb/go-client-mongodb-atlas v0.1.4-0.20200318163834-8b6a5ea6eb67
-	github.com/mongodb/go-client-mongodb-ops-manager v0.0.2-0.20200319100114-648fc659365b
+	github.com/mongodb/go-client-mongodb-atlas v0.1.4-0.20200323160302-12cb92c69adf
+	github.com/mongodb/go-client-mongodb-ops-manager v0.0.2-0.20200324155204-287d08b2e57b
+	github.com/mwielbut/pointy v1.1.0
```

Comment:
any reason to include pointy now? can you please check this? will it compile if we remove this line?

Reply:
I think I'm getting this on master as well, so may as well 🤷‍♂

Reply:
`go mod tidy` does not clean pointy but the application compiles without it. I do not how ponty went there 🤦‍♂

Reply:
If I remove ponty from go.mod and then run `make fmt check build`, ponty is again inside go.mod

Reply:
Found the way, you added it to a fixture